src/Common/src/Microsoft/Win32/SafeHandles/SafeEvpCipherCtxHandle.Unix.cs

```diff
@@ -37,11 +37,11 @@ internal static SafeEvpCipherCtxHandle Create()
 
             try
             {
-                memPtr = Marshal.AllocHGlobal(Interop.libcrypto.EVP_CIPHER_CTX_SIZE);
+                memPtr = Marshal.AllocHGlobal(Interop.Crypto.EVP_CIPHER_CTX_SIZE);
```

Comment:
I personally like making less chatty Create/Destroy shims like what I did for HMAC, but if you guys think 1:1 with GetSize and Alloc/FreeHGlobal is better, I won't push back, but then you should probably switch my HMAC code to match this pattern...